README.md

```diff
@@ -12,7 +12,13 @@ Get notified when a DOM element enters or exits the viewport. A small (~1.9kb gz
 
 ## Installation
 
-Either download the [latest release](https://raw.githubusercontent.com/camwiegert/in-view/master/dist/in-view.min.js) and include it in your markup or install with [npm](http://npmjs.com/package/in-view):
+Either download the [latest release](https://raw.githubusercontent.com/camwiegert/in-view/master/dist/in-view.min.js) and include it in your markup:
+
+```html
+<script src="https://raw.githubusercontent.com/camwiegert/in-view/master/dist/in-view.min.js"></script>
```

Comment:
(sorry, I glossed over this, good catch)

CDNjs is probably fine, never submitted to them before. I wonder if they handle versioning via github tags?

We could also use rawgit:

https://rawgit.com/camwiegert/in-view/master/dist/in-view.min.js <------ throttles on "excessive" use; recommended for dev use

https://cdn.rawgit.com/camwiegert/in-view/master/dist/in-view.min.js  <---- this should be versioned maybe by commithash, because the CDN will cache the link forever; recommended for prod use

Thoughts?